### ISSUE TITLE
linter: false positives for label-has-associated-control

### ISSUE THREAD

**Issue by jelly** [oxc-project/oxc#7849]
### What version of Oxlint are you using?

0.15.0 / main

### What command did you run?

oxlint --import-plugin --jsx-a11y-plugin -D label-has-associated-control

### What happened?

In our projects we currently run eslint with this rule enabled but it doesn't warn us about these issues while with oxlint it does. 

```
  × eslint-plugin-jsx-a11y(label-has-associated-control): A form label must be associated with a control.
     ╭─[src/app.tsx:173:13]
 172 │         <Page>
 173 │             <FilesContext.Provider value={{ addAlert, cwdInfo }}>
     ·             ─────────────────────────────────────────────────────
 174 │                 <WithDialogs>
     ╰────
  help: Either give the label a `htmlFor` attribute with the id of the associated control, or wrap the label around the control.
```
Code: https://github.com/cockpit-project/cockpit-files/blob/13/src/app.tsx#L173

```
  ⚠ eslint-plugin-jsx-a11y(label-has-associated-control): A form label must have accessible text.
    ╭─[src/ContainerDetails.jsx:66:53]
 65 │                         <DescriptionListTerm>{_("Created")}</DescriptionListTerm>
 66 │                         <DescriptionListDescription><utils.RelativeTime time={container.Created} /></DescriptionListDescription>
    ·                                                     ───────────────────────────────────────────────
 67 │                     </DescriptionListGroup>
    ╰────
  help: Ensure the label either has text inside it or is accessibly labelled using an attribute such as `aria-label`, or `aria-labelledby`. You can mark more attributes as
accessible labels by configuring the `labelAttributes` option.
```
